assets/js/atomic/blocks/product-elements/sku/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface Attributes {
```

Comment:
Should this have a more specific name to indicate that it's the Attributes of a Block or is it clear enough?